src/docs/self-hosted/releases.mdx

```diff
@@ -23,7 +23,7 @@ We encourage everyone to regularly update their Sentry installations to get the
 To upgrade, all you need to do is download or check out the version of onpremise repository you want, replace your existing folder's contents with that, and then run `./install.sh`.
 
 <Alert title="Configuration Updates" level="info">
-We may have some updated configuration, especially for new features, so always check the example config files under the [sentry directory](https://github.com/getsentry/onpremise/blob/master/sentry/) and see if you need to update your existing configuration. We do our best to automate critical configuration updates, but you should always check your configs during upgrades.
+We may have some updated configuration, especially for new features, so always check the example config files under the <Link to="https://github.com/getsentry/onpremise/blob/master/sentry/">sentry directory</Link> and see if you need to update your existing configuration. We do our best to automate critical configuration updates, but you should always check your configs during upgrades.
```

Comment:
Ftr, afaict [the docs](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/) suggest using `Link` for internal links and `a` for external.

Reply:
We should probably do an audit for these as we use them pretty liberally for now.